ReactAndroid/src/main/java/com/facebook/react/uimanager/AccessibilityDelegateUtil.java

```diff
@@ -99,6 +102,13 @@ public static AccessibilityRole fromValue(@Nullable String value) {
     }
   }
 
+  // State constants for states which have analogs in AccessibilityNodeInfo
+
+  private static final String STATE_ENABLED = "enabled";
+  private static final String STATE_SELECTED = "selected";
+  private static final String STATE_CHECKED = "checked";
+  private static final String STATE_HAS_POPUP = "hasPopup";
```

Comment:
I think `haspopup` is probably not a state but a relationship. This property comes from web's ARIA spec, where the accepted value must match the role of the popup container - https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup

Reply:
I could support removing this as a state, and handling it in a accessible relationships PR, but then we should submit a PR to remove it from #24095 too.

Reply:
sounds good